public/js/scripts.js

```diff
@@ -183,24 +183,22 @@ function clearQuestions() {
 
 //shows/hides new question container
 function toggleQuestionForm() {
-	$(".newQuestionContainer").slideToggle("slow", function() {
-	    //when the animation is done playing, focuses the keyboard on the textarea
-	    $("#newQuestionTextarea").focus();
-	});
+    if(deviceType != 'desktop') {
```

Comment:
now on desktop the new question container is permanently at the top of the page, so clicking on the new question button simply selects the text area for you if you didn't see it at the top.